### ISSUE TITLE
Transfer Learning with word2vec?

### ISSUE THREAD

**Issue by cemoody** [karpathy/neuraltalk#10]
Hi Andrej & Fei-fei,
I've been playing around with this and reading through the code -- many thanks for making it wonderful code to read! I was under the impression that it used pretrained word vector embeddings from Mikolov et al:
![Image of mikolove ref slide](http://i.imgur.com/Lbx7mxT.png)

....but I don't see any evidence in the code where these vectors are loaded in. Are the word embeddings learned from scratch or are they in fact initialized in some way? 

Many thanks!
chris moody

**Comment by karpathy** [comment]
Yep! I took them out and didn't see a huge decrease in performance, so I decided to get rid of it for NeuralTalk for simplicity.
